app/components/publications/publication-sidebar.hbs

```diff
@@ -36,31 +37,65 @@
               </div>
             {{/if}}
           </div>
+
+          {{!-- Status --}}
           <Publications::PublicationStatusSelector
                   @publicationStatus={{this.publicationStatus}}
                   @onChange={{this.setPublicationStatus}}
+                  @disabled={{or this.publicationStatus.isPublished this.publicationStatus.isWithdrawn}}
           />
-          <div class="auk-form-group">
-            <WebComponents::AuLabel for="blockInput">{{t "regulation-type"}}</WebComponents::AuLabel>
-            <WebComponents::AuPowerSelect @options={{this.regulationTypes}}
-                                          @isSearchable={{false}}
-                                          @onchange={{this.setRegulationType}}
-                                          @selected={{@publicationFlow.regulationType}}
-            />
-          </div>
-          <div class="auk-form-group">
-            <WebComponents::AuLabel for="blockInput">{{t "kind-of-publication"}}</WebComponents::AuLabel>
-            <WebComponents::AuPowerSelect @options={{this.publicationModes}}
-                                          @isSearchable={{false}}
-                                          @onchange={{this.setPublicationMode}}
-                                          @selected={{this.publicationFlow.mode}}
-            />
-          </div>
-          <div class="auk-form-group">
-            <Publications::UrgencyLevelCheckbox
-                    @urgencyLevel={{this.publicationFlow.urgencyLevel}}
-                    @onChange={{this.setUrgencyLevel}} />
-          </div>
+
+          {{!-- TODO: do the same for isPaused --}}
+          {{#if this.publicationStatus.isWithdrawn}}
+            <div class="auk-form-group">
+              <WebComponents::AuLabel>
+                {{t "withdrawn-on"}}
+              </WebComponents::AuLabel>
+              <div class="auk-u-muted">
+                {{moment-format
+                    this.publicationFlow.closingDate
+                    "DD-MM-YYYY"
+                }}
+              </div>
+            </div>
+          {{/if}}
+
+          {{!-- Type regelgeving --}}
+          {{#if (not this.publicationStatus.isWithdrawn)}}
```

Comment:
`{{#if (not ...` kan eenvoudiger geschreven worden als [unless](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=unless). Pas je deze (en anderen in deze PR) aan? Voor complexere condities behouden we wel een gewone `if`. Anders wordt het moeilijker om te interpreteren.